src/main.cpp

```diff
@@ -68,7 +68,7 @@ static void drawFrame(void *arg)
     raylib::core::Window::clear();
     {
         raylib::core::scoped::Mode3D mode3D(*camera);
-        // testingModel.draw(pos, rotationAxis, -90, scale, raylib::core::Color::RED);
+        testingModel.draw(pos, rotationAxis, -90, scale, raylib::core::Color::RED);
```

Comment:
Forgot to remove debug ?

Reply:
It will be removed later anyway when merging the branch on the game or on the Worlds.

Reply:
(With this line commented it made the CI failed because of the unused values above)